go.mod

```diff
@@ -2,14 +2,18 @@ module github.com/libp2p/go-libp2p-kbucket
 
 require (
 	github.com/ipfs/go-ipfs-util v0.0.1
-	github.com/ipfs/go-log v1.0.3
+	github.com/ipfs/go-log v1.0.4
 	github.com/jbenet/goprocess v0.1.4
-	github.com/libp2p/go-libp2p-core v0.5.1
-	github.com/libp2p/go-libp2p-peerstore v0.2.3
+	github.com/libp2p/go-libp2p-asn-util v0.0.0-20200528110405-70ea36266519
```

Comment:
I don't feel super strongly about release vs commit, although it seems a bit silly to not do a release of a thing that will be in production just because it's "not done yet". If we're going to do this I'd like to see the commit in master though, otherwise if we squash, force push or otherwise delete this commit then applications like go-ipfs might not build anymore.

@Stebalien thoughts?

Reply:
Agreed. If we're putting this into production, we shouldn't be working off of branches.

Reply:
Let's just merge that. We don't need to figure out a way to update the ASN DB right now, that's a future concern.